### PR TITLE
Jarvis profile validated, updated after technical midterm

### PR DESCRIPTION
**Pull request by Apivato**

### CODE REVIEW

**Review by jarviscanada — APPROVED** [review]
Give a quick once-over with Grammarly, there are one or two capitalization (python vs Python, etc) or word-choice errors ("then" in the second sentence is not grammatically correct), but otherwise looks good to me. Please push to Master.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
+
+**University of Toronto Scarborough (2015-2016)**, Environmental Science, Department of Physical & Environmental Science
+- 3rd place in UTSC's FinanceHacks Hackathon
+
+
+## Miscellaneous
+- Licensed International Tawkwondo Federation Black Belt Instructor (2018)
+- French Billingual High school Diploma (2015)
+- Certified Standard First Aid & CPR/AED Level C (Canadian Red Cross, 2019)
+- fluent in French and knowledgable in Spanish
```
Comment (jarviscanada) [inline README.md]: Capitalization

File: README.md [hunk 2]
```diff
@@ -1,8 +1,60 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Amos Pivato . Jarvis Consulting
 
+My name is Amos Pivato, and I am a Toronto Metropolitan University Computer Science graduate with experience working in Data science and automation.  I then discovered the world of AI and machine learning and decided that I wanted to leverage my skills with these tools to help build solutions that could help improve the way we live our everyday lives.  Since then, I've worked in Python, Java, and VBA to build automated reporting and information consolidating tools, as well as developed applications from the ground up.  All of this practice has allowed me to grow my skills and proficiencies even further and become a confident and reliable back-end developper.
```
Comment (jarviscanada) [inline README.md]: "...developed applications from the ground up" seems a bit out of place. What sort of applications? Are the reporting and info tools not built from the ground up? Be more specific if possible.

**Review by jarviscanada — APPROVED** [review]
Looks mostly good, but please run it through Grammarly then proofread to catch the various spelling, capitalization, formatting, and grammar errors. Also see the few comments I've made.

File: README.md (hunk 2 above)
Comment (Apivato) [reply README.md]: correct and ran through gramarly

File: README.md (hunk 1 above)
Comment (Apivato) [reply README.md]: correct and ran through gramarly

**Pull request merged**